instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/AwsSdkInstrumenterFactory.java

```diff
@@ -31,7 +31,11 @@
 import software.amazon.awssdk.core.interceptor.ExecutionAttributes;
 import software.amazon.awssdk.core.interceptor.SdkExecutionAttribute;
 
-final class AwsSdkInstrumenterFactory {
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public final class AwsSdkInstrumenterFactory {
```

Comment:
Sorry, I don't understand here very much. Could you explain more about adjustment of visibility here? BTW, if necessary, add same comment on https://github.com/laurit/opentelemetry-java-instrumentation/blob/645e3e3c8658c9231c7327d035a4773743236dc1/instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/SqsAdviceBridge.java#L8

Reply:
This class is used by `AwsSdkTelemetry` that is in a different package. This comment is enforced by tooling on public classes in `internal` package. It is not needed on `SqsAdviceBridge` as it is not in `internal` package.